WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -56,7 +56,12 @@ struct WooShippingCreateLabelsView: View {
                         WooShippingServiceView(viewModel: shippingService)
                             .padding(.horizontal, -16)
                     } else {
-                        WooShippingPackageAndRatePlaceholder()
+                        if let storeOptions = viewModel.storeOptions {
```

Comment:
This is now much cleaner and easier to review 🥳
I put the check for the storeOptions just here so that just this view is not showed before we get storeOptions so later on we can do other changes if needed 🕺 @rachelmcr